Silicon/Apple/AppleSiliconPkg/Drivers/AppleAicDxe/AicV2/AppleAicV2Dxe.c

```diff
@@ -14,10 +14,10 @@
  */
 
 #include <Library/AppleAicLib.h>
-#include <Include/libfdt.h>
```

Comment:
This pops up a good bit in the codebase, some places you leave both the FDT and ADT stuff in, and other times you leave only the ADT based code in, this is going to cause confusion down the line on which will be supported (the ideal is to support both scenarios and dynamically pick which based on our initial conditions with precedence going to ADT, however this is going to need a persistent boolean that survives between UEFI phases which as we've established sucks to do so I'd just remove any FDT stuff completely for now that hasn't transitioned to ADT, it can always be added back in later)